script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -462,27 +462,27 @@ It's as though he were grinning in agreement with Arcueid's provocation.
 [sha:b8aaf187460b84ead817cb5a6b19a3c96c926449]{
 -- Page 548, Offset 10071.
 -- 「最も強きものとは千変にして<偏|かたよ>らぬもの。
-"The strongest ones are those who continue to adapt and do not remain locked in their ways.
+"Those that are strong... are those that through vicissitudes remain unfaltering.
 }
 [sha:52f1281e1a8e19a264a2c690652cc29cab9f9ace]{
 -- Page 548, Offset 10072.
 -- ……貴様はまさにソレだ。そこまで衰退していながら、まだおれを上回る余力がある。
-...You are exactly that. Even diminished as you are, your remaining power still surpasses my own.
+And that is truly what you are... Though diminished your power may be... above mine it is nonetheless.
 }
 [sha:0cf50eb40e3ac194055940c470a381326109bc97]{
 -- Page 548, Offset 10073.
 -- あらゆる呪い、あらゆる状況に対応する原初の<一|いち>―――並の死徒ではかなうまい」
-A primordial one that deals with every curse and every situation――― No ordinary Dead Apostle can possibly compare."
+One who can with all curses deal; through all situations adapt: The Original One―――an ordinary Dead Apostle stands no chance..."
 }
 [sha:6fccdd5afb420650f022fe55982600c584aa0ba3]{
 -- Page 549, Offset 10074.
 -- 空気が歪む。
-The atmosphere distorts.
+The air distorts.
 }
 [sha:dfeb0e6fd3142597c92bf065b5e4310592366e77]{
 -- Page 549, Offset 10075.
 -- 病想の青い熱が、地下の気温を下げていく。
-The blue fever of ill thoughts lowers the temperature in the crater.
+In thoughts ill does the blue blaze start dropping the temperature of the crater.
```

Comment:
This one still rubs me the wrong way a bit, I think partly because it uses Vlov's speech pattern but is actually narration, and partly because I'm not 100% on what 病想 is really trying to get across. Is it that the flames are products / called forth because of Vlov's ill will? If it's that reading, malicious or malevolent might be more clear. If they are themselves something that is sickening then maybe pestilential or similar?